packages/teleport/src/Databases/AddDatabase/AddDatabase.tsx

```diff
@@ -333,25 +339,106 @@ const generateDbStartCmd = (
   }
 };
 
+// options is the list of db options used
+// for a dropdown menu.
+//
+// Modifying this list also requires modifying:
+// - enum DatabaseOption
+// - function getDatabaseOption
```

Comment:
I feel like there is a much better less error prone way to do this, i'm still thinking :sweat:

Reply:
resolved by removing dropdown selector